Extensions/Effects/crt-pixi-filter.js

```diff
@@ -1,13 +1,19 @@
 gdjs.PixiFiltersTools.registerFilterCreator('CRT', {
   makePIXIFilter: function(layer, effectData) {
     var crtFilter = new PIXI.filters.CRTFilter();
-
+    crtFilter._animationTimer = 0;
     return crtFilter;
   },
   update: function(filter, layer) {
-    if (filter.animated) {
-      filter.time += layer.getElapsedTime() / 1000;
-      filter.seed = Math.random();
+    if (filter.animationSpeed !== 0) { 
+      filter.time += layer.getElapsedTime() / 100 * filter.animationSpeed;
```

Comment:
`/ 100` looks like a bug. Avoid magic numbers, prefer doing:
`/ 1000 * 10 * filter.animationSpeed`

and add a comment:
`// Multiply by 10 so that the default value shows use a sensible speed`